docs/markdown/star_formation.md

```diff
@@ -0,0 +1,33 @@
+# Star Formation Recipe 
+
+## Checking for Jeans Violation
+The star formation recipe is implemented via the Stochastic Stellar Population specialisation. Every cell is checked for Jeans violation and star particles are added if $\lambda_J = c_s/\sqrt{G\rho} < J \cdot dx$, where J is $0.5$. 
+
+Adding star particle to every Jeans violating cell can lead to a large number of small mass stars which can be cumbersome. To alleviate this issue, we control the star formation rate through $\epsilon_{\rm ff}$ and $\epsilon_{*}$. $\epsilon_{\rm ff}$ is the efficiency per freefall time given by $\frac{\dot{M}_{*} \cdot t_{\rm ff}}{M_{\rm cell} \cdot dt}$, while $\epsilon_{*}$ is the fraction of cell mass that gets converted into stars. 
+
+There are two equivalent ways of converting a certain mass from cell into stars. 
+
+In the first, we assume that every cell that is Jeans unstable definitely forms a star. In this scenario, $M_{*} = \epsilon_{\rm ff} \cdot dt \cdot M/t_{\rm ff}$ is the mass that gets converted into stars from a cell mass of M. This will lead to the issue mentioned earlier, that is of a large number of stars being created in a galaxy-scale simulation.
```

Comment:
Can we delete this paragraph? Since we don't use this method, I think it is better to remove it for the sake of clarity.

Reply:
I think it's fine to leave it, at least for now.